spec/dpn/client/agent/connection_spec.rb

```diff
@@ -33,11 +30,9 @@
   end
 
   it "returns errors" do
-    error_url = File.join url, "error", "/"
```

Comment:
Once upon a time all DPN urls were to end in a slash, but that's no longer how the spec is defined.  Good to see that changed here.